### PR TITLE
bug(product.rb/ bigcommerce_product_agent): fix error logging bug

### PR DESCRIPTION
**Pull request by Jhalter5Stones**
fix error logging bug

### CODE REVIEW

**Comment by Jhalter5Stones** [comment]
@mrogers-5s @Ken-Michalak Ready for review

https://trello.com/c/ijNLVm8y/587-dsbat-identify-products-when-sync-events-fail

File: lib/client/product.rb [hunk 1]
```diff
@@ -37,10 +37,10 @@ def update_batch(payload, params={})
                   e.response[:status],
                   'update product',
                   'Failed to update product batch',
+                  payload.map { |p|
+                    { sku: p[:sku] }
+                  },
                   {
-                    product_identifiers: payload.map { |p|
-                      { id: p[:id], sku: p[:sku] }
-                    },
```
Comment (mrogers-5s) [inline lib/client/product.rb:43]: @Jhalter5Stones Why are we removing `id` here?

**Review by mrogers-5s — CHANGES_REQUESTED** [review]

File: lib/client/product.rb (hunk 1 above)
Comment (Jhalter5Stones) [reply lib/client/product.rb:43]: @mrogers-5s id was null for some reason. Maybe the name is different at this point?

Comment (mrogers-5s) [reply lib/client/product.rb:43]: The ID _may_ be `null` for new products, but it shouldn't be `null` for existing products.

Comment (Jhalter5Stones) [reply lib/client/product.rb]: @mrogers-5s I think I fixed this. Let me know if it looks right now.

**Review by mrogers-5s — APPROVED** [review]

**Review by Ken-Michalak — APPROVED** [review]

**Pull request merged**